core/src/main/scala/org/apache/spark/sql/delta/metering/DeltaLogging.scala

```diff
@@ -47,9 +52,10 @@ import org.apache.spark.sql.delta.util.JsonUtils
  *  Underneath these functions use the standard usage log reporting defined in
  *  [[com.databricks.spark.util.DatabricksLogging]].
  */
-trait DeltaLogging
-  extends DeltaProgressReporter
-  with DatabricksLogging {
+trait DeltaLogging extends DeltaProgressReporter {
+
+  protected def logConsole(line: String): Unit =
+    LoggerImplementation.activeLogger.logConsole(line)
```

Comment:
is this a mistake ?

Reply:
I needed to add `logConsole` to `DeltaLogging` since `DeltaLogging` doesn't extend `DatabricksLogging` any more.